sk/lessons/basics/enum.md

```diff
@@ -49,11 +49,11 @@ Táto funckia má niekoľko možností použitia, pozrite sa do oficiálnej doku
 
 ### chunk_by
 
-Ak potrebujete rozdeliť kolekciu do menších na základe niečoho iného, než ich veľkosť, použite funkciu `chunk_by`:
+Ak potrebujete rozdeliť kolekciu do menších na základe niečoho iného, než ich veľkosť, použite funkciu `chunk_by`. Ako argumenty funkcia berie kolekciu (Enum) a funkciu - ak sa zmení jej výstup, ukončí sa aktuálna podkolekcia a začne sa vytvárať ďalšia:
 
 ```elixir
-iex> Enum.chunk_by(["one", "two", "three", "four", "five"], fn(x) -> String.length(x) end)
```

Comment:
We did not. I added both because the intention was to show that when calling `Enum.chunk_by/2` the grouping begins anew when the `function` argument changes its returned result. Though using almost-the-same list may not have been the perfect decision to change it again and back-propogate to all other translation branches would be a much bigger pain.